pkg/container/container_replacements.go

```diff
@@ -17,20 +17,20 @@
 package container
 
 import (
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
```

Comment:
NIT: this was v1 without the alias too :)

Reply:
Thanks Andrea, I was having the same impression on this. But it seems that this is now required by our goling:
```
🐱 Testing if golint has been done…
test/ignore_step_error_test.go:28: File is not `goimports`-ed (goimports)
	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
make: *** [Makefile:175: golangci-lint-check] Error 1
```

Reply:
NIT: this was v1 without the alias too :)

Reply:
Good to know, thank you!